backend/Cargo.toml

```diff
@@ -45,4 +46,4 @@ tokio = { version = "1.35.1", features = ["macros", "rt-multi-thread"] }
 tracing = { version = "0.1.40" }
 tracing-subscriber = { version = "0.3.18", features = ["env-filter"] }
 url = { version = "2.5.0" }
-uuid = { version = "1.7.0", features = ["v4"] }
+uuid = { version = "1.7.0", features = ["v4", "v7"] }
```

Comment:
Would like to just have one of `v4` & `v7`, can we merge an update to move current ones to `v7` before this?